tools/build.py

```diff
@@ -174,22 +174,26 @@ def check_autotools():
 """Check for fusesoc
 """
 def check_fusesoc(min_version):
-    found_version = check_program("fusesoc")
+    found_version_str = check_program("fusesoc")
+    found_version = version.parse(found_version_str)
 
-    if version.parse(found_version) < version.parse(min_version):
+    if found_version.is_devrelease:
+        found_version = version.parse(found_version.base_version)
+
+    if found_version < version.parse(min_version):
         fatal("OpTiMSoC requires at least FuseSoC version {}, version {} found. "
               "Use 'pip3 install --upgrade fusesoc' to update."
-              .format(min_version, found_version))
+              .format(min_version, found_version_str))
     else:
-        dbg("Found FuseSoC version " + found_version)
+        dbg("Found FuseSoC version " + found_version_str)
 
 """Check for vivado
 """
 def check_vivado(min_version):
     ver_str = check_program('vivado', '-version')
     # XXX: most likely doesn't cover service releases, extend if we see one of
```

Comment:
Please remove that now-outdated comment.

Reply:
Done in force push